modules/configuring-administration-events-retention-settings.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// *operating/using-the-administration-events-dashboard.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-administration-events-retention-settings_{context}"]
+= Configuring administration events retention settings
+
+By configuring the retention settings, you can determine when the administration events expire.
+
+.Procedure
+
+. In the {product-title-short} portal, navigate to *Platform Configuration ->  System Configuration*. You can configure the following setting for administration events:
+
+* *Administration events retention days*: The number of days to retain your administration events. By specifying the number of days, you can control when these events expire. This is important for managing your administration events and making sure that you retain the information for the desired duration.
++
+[NOTE]
+====
+By default, the administration events are retained for 4 days. After the retention period expires, only individual events within the administration events dashboard expire.
```

Comment:
One point on the retention period: each event is associated with the created time and last occurred time.
An event expires when the _last occurred time_ has reached the retention period, not when it was created.

So, essentially what it means for customers: if the event is still occurring (e.g. say an image cannot be scanned for a longer period), and the last time of occurrence is lower to the retention period, it will _not_ be deleted. This might be good to explicitly call out to not have people wonder "why is the event not deleted when it was created X days ago?".

Reply:
As discussed in chat, I will go ahead with the following content:
"By default, the administration events are retained for 4 days. The retention period for these events is determined by the time of the last occurrence and not by the time of creation. This means that an event expires and is deleted only if the time of the last occurrence exceeds the specified retention period."